Sync both the content and style of `shutdown -h` and `man shutdown`.

http://bugs.gentoo.org/91724

Index: man/shutdown.8
===================================================================
--- man/shutdown.8.orig
+++ man/shutdown.8
@@ -9,11 +9,11 @@ shutdown \- bring the system down
 .\"{{{  Synopsis
 .SH SYNOPSIS
 .B /sbin/shutdown
+.RB [ \-akrhPHfFnc ]
 .RB [ \-t
 .IR sec ]
-.RB [ \-arkhncfFHP ]
 .I time
-.RI [ warning-message ]
+.RI [ "warning message" ]
 .\"}}}
 .\"{{{  Description
 .SH DESCRIPTION
@@ -39,11 +39,6 @@ entries for these runlevels in the file 
 .IP "\fB\-a\fP
 Use \fB/etc/shutdown.allow\fP.
 .\"}}}
-.\"{{{  -t sec
-.IP "\fB\-t\fP \fIsec\fP"
-Tell \fBinit\fP(8) to wait \fIsec\fP seconds between sending processes the 
-warning and the kill signal, before changing to another runlevel.
-.\"}}}
 .\"{{{  -k
 .IP \fB\-k\fP
 Don't really shutdown; only send the warning messages to everybody.
@@ -56,20 +51,14 @@ Reboot after shutdown.
 .IP \fB\-h\fP
 Halt or power off after shutdown.
 .\"}}}
-.\"{{{  -H
-.IP \fB\-H\fP
-Modifier to the -h flag.  Halt action is to halt or drop into boot
-monitor on systems that support it.  Must be used with the -h flag.
-.\"}}}
 .\"{{{  -P
 .IP \fB\-P\fP
 Halt action is to turn off the power.
 .\"}}}
-.\"{{{  -n
-.IP \fB\-n\fP
-[DEPRECATED] Don't call \fBinit\fP(8) to do the shutdown but do it ourself.
-The use of this option is discouraged, and its results are not always what
-you'd expect.
+.\"{{{  -H
+.IP \fB\-H\fP
+Halt action is to halt or drop into boot monitor on systems that
+support it.
 .\"}}}
 .\"{{{  -f
 .IP \fB\-f\fP
@@ -79,18 +68,29 @@ Skip fsck on reboot.
 .IP \fB\-F\fP
 Force fsck on reboot.
 .\"}}}
+.\"{{{  -n
+.IP \fB\-n\fP
+[DEPRECATED] Don't call \fBinit\fP(8) to do the shutdown but do it ourself.
+The use of this option is discouraged, and its results are not always what
+you'd expect.
+.\"}}}
 .\"{{{  -c
 .IP \fB\-c\fP
 Cancel an already running shutdown. With this option it is of course
 not possible to give the \fBtime\fP argument, but you can enter a
 explanatory message on the command line that will be sent to all users.
 .\"}}}
+.\"{{{  -t sec
+.IP "\fB\-t\fP \fIsec\fP"
+Tell \fBinit\fP(8) to wait \fIsec\fP seconds between sending processes the 
+warning and the kill signal, before changing to another runlevel.
+.\"}}}
 .\"{{{  time
 .IP \fItime\fP
 When to shutdown.
 .\"}}}
 .\"{{{  warning-message
-.IP \fIwarning-message\fP
+.IP "\fIwarning message\fP"
 Message to send to all users.
 .\"}}}
 .PP
Index: src/shutdown.c
===================================================================
--- src/shutdown.c.orig
+++ src/shutdown.c
@@ -102,7 +102,7 @@ void stopit(int sig)
 void usage(void)
 {
 	fprintf(stderr,
-	"Usage:\t  shutdown [-akrhHPfnc] [-t secs] time [warning message]\n"
+	"Usage:\t  shutdown [-akrhPHfFnc] [-t sec] time [warning message]\n"
 	"\t\t  -a:      use /etc/shutdown.allow\n"
 	"\t\t  -k:      don't really shutdown, only warn.\n"
 	"\t\t  -r:      reboot after shutdown.\n"
